lib/qrCode.js

```diff
@@ -1,7 +1,7 @@
 const apiRequest = require('./apiRequest');
 
-async function getQRCodeUrl(apiSecret, apiKey, amount, cause) { 
-  const apiResponse = await apiRequest(apiSecret, apiKey, "POST", "/transaction/qr-generate", cause?{amount, cause}: {amount})
+async function getQRCodeUrl(amount, cause) { 
+  const apiResponse = await apiRequest(process.env.API_SECRET, process.env.API_KEY, "POST", "/transaction/qr-generate", cause?{amount, cause}: {amount})
```

Comment:
You can read the env variables in called function itself, no need to send from each API call.

Reply:
@ermiyas111